kentikapi/models/user.go

```diff
@@ -2,24 +2,45 @@ package models
 
 import "time"
 
-type GetAllUsersResponse struct {
-	Users []User `json:"users"`
+// User model.
+type User struct {
+	// read-write properties (can be updated in update call)
+	Username     string
```

Comment:
I just checked with Postman, that updating username returns 200, but does not update that field.
I think that I should mark this field as not updatable (read-only), what do you think?

Reply:
I agree, if it cannot be updated then it should be marked properly.